framework/src/base/FEProblemBase.C

```diff
@@ -4428,12 +4431,14 @@ FEProblemBase::adaptMesh()
     else
     {
       _console << "Mesh unchanged, skipping remaining steps..." << std::endl;
```

Comment:
Should of had you take out this extra `std::endl` since we have an explicit `std::flush` in the next line. I/O flush operations can really, really slow you down (relative to the execution of every other computational instruction). We need to keep them to a minimum and use new lines instead of full flushes when we don't need them. People don't tend to notice on their laptops but they can be painfully slow on a NFS system.